examples/webrtc/main.go

```diff
@@ -99,25 +99,27 @@ func main() {
 		}
 	}
 
-	// Set the remote SessionDescription
```

Comment:
My bad, I think I was not clear 😅. These changes should not be needed, we still need to set remote description and local description to set the right state to PeerConnection and spin up the ICE gatherer.

Reply:
Ok I think my latest commit fixed this. Is it right that setRemoteDescription starts ICE gathering and uses the info set by setLocalDescription?

Reply:
Ohhh dear there's a lot to learnXD

Reply:
Ok so I think answer is a SDP that's supposed to be sent to the browser, but is the data at the memory location *peerConnection.LocalDescription() an SDP with updated ICE candidates? so we only have to send `fmt.Println(signal.Encode(*peerConnection.LocalDescription()))` for the user to paste in their browser?

Reply:
Yep, that's right!

Reply:
Cool!